src/main/java/uk/gov/hmcts/divorce/caseworker/event/PaymentMadeFromAwaitingPayment.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.divorce.caseworker.event;
+
+import uk.gov.hmcts.ccd.sdk.api.CCDConfig;
+import uk.gov.hmcts.ccd.sdk.api.ConfigBuilder;
+import uk.gov.hmcts.divorce.ccd.PageBuilder;
+import uk.gov.hmcts.divorce.common.model.CaseData;
+import uk.gov.hmcts.divorce.common.model.State;
+import uk.gov.hmcts.divorce.common.model.UserRole;
+
+import static uk.gov.hmcts.divorce.common.model.State.AwaitingPayment;
+import static uk.gov.hmcts.divorce.common.model.State.Submitted;
+import static uk.gov.hmcts.divorce.common.model.UserRole.CASEWORKER_DIVORCE_COURTADMIN;
+import static uk.gov.hmcts.divorce.common.model.UserRole.CASEWORKER_DIVORCE_COURTADMIN_BETA;
+import static uk.gov.hmcts.divorce.common.model.UserRole.CASEWORKER_DIVORCE_COURTADMIN_LA;
+import static uk.gov.hmcts.divorce.common.model.UserRole.CASEWORKER_DIVORCE_SUPERUSER;
+import static uk.gov.hmcts.divorce.common.model.UserRole.CITIZEN;
+import static uk.gov.hmcts.divorce.common.model.access.Permissions.CREATE_READ_UPDATE;
+import static uk.gov.hmcts.divorce.common.model.access.Permissions.READ;
+
+public class PaymentMadeFromAwaitingPayment implements CCDConfig<CaseData, State, UserRole> {
+
+    public static final String PAYMENT_MADE_FROM_AWAITING_PAYMENT = "payment-made-from-awaiting-payment";
```

Comment:
maybe prefix the event name class name with caseworker

Reply:
Didn't  add as the event is accessible by citizen also. But yeah can update class name